.cci.jenkinsfile

```diff
@@ -0,0 +1,43 @@
+// Documentation: https://github.com/coreos/coreos-ci/blob/main/README-upstream-ci.md
+
+properties([
+    // abort previous runs when a PR is updated to save resources
+    disableConcurrentBuilds(abortPrevious: true)
+])
+
+
+buildPod(runAsUser: 0, memory: "2Gi", cpu: "2") {
+      checkout scm
+      stage("Install packit") {
+        shwrap("""
+          dnf install packit -y
+        """)
+      }
+      stage("Build kdump RPM") {
+        // note: coreos upstream CI only run on x86 so we hardcode the architecture
+        shwrap("""
+          packit build locally --release-suffix coreos.tests
+          mv x86_64/kdump-utils*coreos.tests*.rpm ./kdump-coreos-ci.rpm
+        """)
+      }
+      stash includes: 'kdump-coreos-ci.rpm', name: 'kdump-rpm'
```

Comment:
It's also useful to archive the RPMs so that anyone can also download them and test them locally. This has come in quite handy a few times in rpm-ostree:

https://github.com/coreos/rpm-ostree/blob/94b701f1bcc4ae9067a5dc350e43e28f20459ac7/.cci.jenkinsfile#L15-L16